src/features/currentTodo.ts

```diff
@@ -1,10 +1,17 @@
-import { createSlice } from '@reduxjs/toolkit';
-import { Todo } from '../types/Todo';
+import { createSlice, PayloadAction, Slice } from '@reduxjs/toolkit';
+import { Todo } from '../types';
 
 const initialState = null as Todo | null;
```

Comment:
typing initialState can make the code more readable

```suggestion
type CurrentTodoState = Todo | null;

const initialState: CurrentTodoState = null;
```

Reply:
Yes, indeed. But I still have to specify it through 'as'. Because I still get type incompatibility

![image](https://github.com/user-attachments/assets/17402f9f-c82c-4380-8a44-38ad531e2db1)

Reply:
Ok, I found the solution. But now I don't specify the type of PayloadAction. The code works, but i think it's not quite right to do it this way 🤔
![image](https://github.com/user-attachments/assets/32781d78-4f2d-47c7-aae6-71ae18fde781)

Reply:
You fix it? because your current version of code is correct

Reply:
Yes, I fixed it. But i'm not sure that this code is completely correct. I'd like to see the opinion of the first mentor @dinaminahi